server/jetstream_benchmark_test.go

```diff
@@ -335,12 +338,21 @@ func BenchmarkJetStreamConsume(b *testing.B) {
 								b.Fatalf("Error creating stream: %v", err)
 							}
 
+							var jsStreamLeader nats.JetStreamContext
+							// reassign connection to stream leader only if cluster_size > 1
+							if bc.clusterSize > 1 {
```

Comment:
How about

```
if cl != nil {
 // close initial connection
 // nc = new connection to leader
 // js = new context from new connection
}
```
4 lines code change rather than 50

Reply:
Could even make it into `fun reconnectToStreamLeader()` rather than repeating this logic in each benchmark